docs/deprecated.md

```diff
@@ -207,3 +208,17 @@ import { Command } from 'commander';
 ```
 
 README updated in Commander v9. Deprecated from Commander v9.
+
+## ._args
```

Comment:
Right idea, and I do want an entry in this doc, thanks. I will change this text. Shall I just do that after (if) gets merged, or would you like to incorporate rewrite?

Reply:
Feel free to do it after it is merged, I don't mind.